config.js

```diff
@@ -1,3 +1,5 @@
+var userKey = 'putOwnApiKeyHere'; //get it on https://developers.zomato.com/api
```

Comment:
use eniroment variable to store API key so we can set it up in our hosting and not push it into git, for example `process.env.ZOMATO_API_KEY`